README.md

```diff
@@ -47,8 +47,9 @@ Usage:
   speculate env ROLENAME [flags]
 
 Flags:
-  -a, --account string   Account ID to assume role on (defaults to source account
-  -l, --lifetime int     Set lifetime of credentials in seconds (defaults to 3600 seconds / 1 hour, min 900, max 3600) (default 3600)
+  -a, --account string   Account ID to assume role on (defaults to source account)
+  -h, --help             help for env
```

Comment:
The help flag should be auto-added by the flag library; is it not being added for you when you run `speculate env -h` ?